guides/internals.rst

```diff
@@ -98,7 +98,7 @@ the `testing guide <testing-hypothesis.rst>`_, but there are a couple
 of areas that it's worth specifically highlighting for making changes
 to the engine:
 
-The first is `tests/cover/test_conjecture_engine.py <https://github.com/HypothesisWorks/hypothesis/blob/master/hypothesis-python/tests/cover/test_conjecture_engine.py>`_,
+The first is `tests/conjecture/test_conjecture_engine.py <https://github.com/HypothesisWorks/hypothesis/blob/master/hypothesis-python/tests/conjecture/test_conjecture_engine.py>`_,
```

Comment:
I think we should probably just point to the whole `tests/conjecture/` directory; the file you're naming doesn't exist either and the tests are now spread across multiple files 🙂 

```suggestion
The first is `tests/conjecture/ <https://github.com/HypothesisWorks/hypothesis/tree/master/hypothesis-python/tests/conjecture>`_,
```
I really appreciate your attention to the docs though 🙌